docs/src/free_associative_algebra.md

```diff
@@ -229,6 +229,19 @@ generators of that ideal.
 Since such a Groebner basis is not necessarily finite, one can additionally pass a `reduction_bound`
 to the function, to only compute a partial Groebner basis.
 
+```@docs
+    groebner_basis(g::Vector{FreeAssAlgElem{T}}, reduction_bound::Int = typemax(Int), remove_redundancies::Bool = false) where T <: FieldElement
+
+    normal_form(f::FreeAssAlgElem{T}, g::Vector{FreeAssAlgElem{T}}, aut::AhoCorasickAutomaton) where T
+
+    interreduce!(g::Vector{FreeAssAlgElem{T}}) where T
+```
+
+The implementation uses a non-commutative version of the Buchberger algorithm as described in
+> Xingqiang Xiu,
+> Non-commutative Gröbner Bases and Applications,
+> PhD thesis, 2012.
+
```

Comment:
You can add this as a proper bibtex reference in `doc/oscar_references.bib` and then cite it here with `[LABEL](@cite)`.

Reply:
Sorry, wrong repository.